README.md

```diff
@@ -24,19 +16,15 @@ level of backwards compatibility to the official releases.
 
 **Pull requests with unofficial bug fixes and security patches from the community are encouraged and welcome!**
 
-### Versioning
-
-Though Magento does __not__ follow [Semantic Versioning](http://semver.org/) we aim to provide a workable system for
-dependency definition. Each Magento `1.<minor>.<revision>` release will get its own branch (named `1.<minor>.<revision>.x`)
-that will be independently maintained with upstream patches and community bug fixes for as long as it makes sense
-to do so (based on available resources). For example, Magento version `1.9.4.5` was merged into the `1.9.4.x` branch.
-
 ## Requirements
 
-- PHP 7.3+ (PHP 8.0 is supported)<br/>
-`mcrypt` extension needs to be disabled to fallback on `mcryptcompat` and `phpseclib`. 
+- PHP 7.3+ (PHP 8.0 is supported)
 - MySQL 5.6+ (8.0+ recommended) or MariaDB
 
+
+- PHP extension `intl` <small>since 1.9.4.19 & 20.0.17</small>
+- Command `patch` (v2.7+) <small>since 1.9.5.0 & 20.1.0</small>
```

Comment:
should we write about gpatch for homebrew?

Reply:
Was not sure about it.

Put it to requirements or add section for mac users?